### PR TITLE
Set milestone test flag to agama_base.pm

### PR DESCRIPTION
**Pull request by coolgw**
Set milestone test flag to agama_base.pm

- Related ticket: https://progress.opensuse.org/issues/131483
- Needles: na
- Verification run: https://openqa.opensuse.org/tests/3420599#

### CODE REVIEW

**Comment by sofiasyria** [comment]
LGTM

**Comment by rakoenig** [comment]
LGTM

File: lib/yam/agama/agama_base.pm [hunk 1]
```diff
@@ -33,4 +33,8 @@ sub post_run_hook {
     ensure_serialdev_permissions;
 }
 
+sub test_flags {
+    return {fatal => 1, milestone => 1};
```
Comment (lemon-suse) [inline lib/yam/agama/agama_base.pm:37]: I think we don't need to set milestone since we haven't such requirement for a snapshot here.

Comment (coolgw) [reply lib/yam/agama/agama_base.pm:37]: I just use this flag to stop failed case, not used for make snapshot here.

Comment (lemon-suse) [reply lib/yam/agama/agama_base.pm:37]: If just want to stop when failed, we can just set 'fatal=1', 'milestone' is for a snapshot for the current test environment.

Comment (jknphy) [reply lib/yam/agama/agama_base.pm:37]: afir milestone shouldn't be needed. seems that PR need rebase now.

Comment (coolgw) [reply lib/yam/agama/agama_base.pm]: @jknphy PR arleady rebase and VR done.

**Pull request merged**